src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Events/EventPage.razor

```diff
@@ -15,6 +16,13 @@
     <OfficialList />
 </MudPaper>
 
+<MudDivider />
+
+<MudPaper>
+   <CompetitionTable></CompetitionTable>
```

Comment:
One of the ideas I had about a naming convention of the Components was to ensure each file contains the name of the programmatical functionality it provides. This would hopefully increase the readability and ease of reference of the code base. For example:

└── Competitions
      ├── CompetitionForm.razor - responsible for exposing user interface for manipulation of the FormModel.
      ├── CompetitionFormModel.cs - UI logic over the Domain Entity
      ├── CompetitionCRD.razor - UI for create, read and delete operations over the stored data of the Domain Entity                                      
      └── CompetitionUpdate.razor - UI for update by utilizing the Form